### DISCUSSION TITLE
where do i change permissions to write/read/delete folders?

### DISCUSSION THREAD

**Discussion by jviniciuspassos** [ciur/papermerge#328] [Q&A] [answered]
I'm new to papermerge, and i saw a issue #323 report about the permissions to create/read/delete folders and didn't found this screen in webapp, i'm using version 1.5.5. Is this kind of configuration only available to version 2.0+?

**Comment by ciur** [comment, marked as answer]
Yes, roles and screen your are mentioning about are introduced in 2.0
